README.md

```diff
@@ -18,62 +18,28 @@ Install the following package into you class library and platform-specific proje
 
 ## How to use
 
-* For those ViewModels which need validation, implement `ISupportsValidation`
-* Add validation rules to the ViewModel
-* Bind to the validation rules in the View
+* For those ViewModels which need validation, implement `ISupportsValidation`.
```

Comment:
`For those ViewModels` should he 

`For ViewModels`

The those is prosessive pronoun which is likely not what we want here.